daemon/manager/manager.go

```diff
@@ -101,17 +101,23 @@ func Execute(configs *options.BootstrapConfigs) error {
 	log.Info("command Start cni successful")
 	defer cniInstaller.Stop()
 
-	setupCloseHandler()
+	*configs.Status = setupCloseHandler()
+	log.Printf("bpfLoader.Restart %v", *configs.Status)
 	return nil
 }
 
-func setupCloseHandler() {
+func setupCloseHandler() int {
```

Comment:
This function seems to be used to handle os signal and it's weired to get `status` and return here.

Reply:
it doesn‘t matter, i can move func GetDaemonset out